### ISSUE TITLE
Update documentation | Start here

### ISSUE THREAD

**Issue by demilolu** [Call-for-Code-for-Racial-Justice/Legit-Info#99]
## Background

This issue provides directions on how you can contribute to Legit-Info by updating our documentation. Please keep this issue open.

You can make both technical and non-technical updates to documentation. 

Non-technical include things like grammar and basic. These are welcome, but please don't make

## Tasks

- [ ] Check out the open documentation issues
- [ ] Read & follow the instructions in the [Getting Started section ](https://github.com/Call-for-Code-for-Racial-Justice/Legit-Info#getting-started)of the readme.md to set the solution up  

If any directions are unclear, outdated, and are not addressed in [open documentation issues](https://github.com/Call-for-Code-for-Racial-Justice/Legit-Info/issues?q=is%3Aopen+is%3Aissue+label%3Adocumentation): 
- [ ] create a new issue using the documentation template
- [ ] label it 'hacktobefest', 'documentation' assign the issue to yourself,
- [ ] Discuss your changes in the issue with @upkarlidder @tonypearson @demilolu 

Once your documentation idea is approved
- [ ] update the readme.md file with your changes
- [ ] submit a pull request to main branch
- [ ] PR merged by Legit-Info team
- [ ] Congratulations! 

If you are unfamilar with pull requests, committing code on Github, we recommend you check out our free course on open source: ibm.biz/opensourcecourse
